packages/block-editor/src/components/iframe/content.scss

```diff
@@ -22,8 +22,11 @@
 }
 
 .block-editor-iframe__html {
+	$beizer: cubic-bezier(0.65, 0, 0.45, 1);
+	$duration: 0.5s;
+	border: 0 solid $gray-300;
```

Comment:
Why `0` for border width?

Reply:
It helps smooth the transition, as it can grow from 0, instead of applying a border.